clippy_lints/src/non_expressive_names.rs

```diff
@@ -133,6 +153,15 @@ impl<'a, 'tcx, 'b> SimilarNamesNameVisitor<'a, 'tcx, 'b> {
         if interned_name.chars().any(char::is_uppercase) {
             return;
         }
+        if interned_name.chars().all(|c| c.is_digit(10) || c == '_') {
+            span_lint(
+                self.0.cx,
+                JUST_UNDERSCORES_AND_NUMBERS,
```

Comment:
this should be `JUST_UNDERSCORES_AND_DIGITS`.

You can run clippy's tests with `cargo clippy`

Reply:
Nice catch, I thought that they are includded in cargo test (or maybe I missed the output)

Reply:
sorry, I meant `cargo test`..

Also, I didn't catch it, travis did ;)